GameData/Skopos/Contracts/ExperimentalComsats/intermittent_transpacific_tv.cfg

```diff
@@ -13,7 +13,7 @@ CONTRACT_TYPE {
   sortKey = 020
   description = <b>Program: Experimental Telecommunications Satellites (Skopos Beta)<br>Type: <color=green>Required</color></b><br><br>The development of artificial satellites opens the possibility of applying microwave radio relay technology to transoceanic links, using a “microwave repeater in the sky”. The objective for this experiment is to answer the critical questions, leaving until a later round of design the optimization of trade-offs and the development and construction of a commercial operating system.
   synopsis = Provide transpacific television for three months with 1% availability (~15 min per day).
-  notes = Earth stations with cutting-edge antennas have been built in the Kashima prefecture (Japan), and at Point Mugu (California). It should be possible to establish a transpacific link with a very small satellite.
+  notes = Earth stations with cutting-edge antennas have been built in the Kashima prefecture (Japan) and at Point Mugu (California). It should be possible to establish a transpacific link with a very small satellite.
```

Comment:
There is an asymmetry in geographical terminology here that tickles my OCD.  "Kashima prefecture" is a subdivision of Japan.  Similarly, "California" is a subdivision of the USA.  I'd prefer if we always used the name of the country in the parentheses, otherwise it looks a tad US-centric.  For instance, I'd write "in the Kashima prefecture (Japan) and at Point Mugu, California (USA)".  There are probably other instances in this file.